cli/vendor/github.com/crewjam/go-cloudformation/schema.go

```diff
@@ -10385,7 +10385,7 @@ type EC2SecurityGroupRule struct {
 	// Security Group to allow access. You can use the Ref intrinsic function
 	// to refer to the logical ID of a security group defined in the same
 	// template.
-	SourceSecurityGroupIdXXSecurityGroupIngressXOnlyX *StringExpr `json:"SourceSecurityGroupId (SecurityGroupIngress only),omitempty"`
+	SourceSecurityGroupIdXXSecurityGroupIngressXOnlyX *StringExpr `json:"SourceSecurityGroupId,omitempty"`
```

Comment:
Are you sending this to the library? We should have a unit test or something, so that we don't override this change by mistake with a `glide update`.

Reply:
This might be a problem for tools integrating i2kit. Not sure what's the right solution for this, if a fork, or a serialization override. we should test that before merging it.